dashboard/app/views/levels/editors/_aichat.html.haml

```diff
@@ -1,3 +1,3 @@
 = render partial: 'levels/editors/fields/aichat_fields', locals: {f: f}
-
+= render partial: 'levels/editors/fields/instructions_area', locals: {f: f}
```

Comment:
No video levels needed - correct?

Reply:
I think so -- I chatted with Amy BW about that, who said videos in their own levels would be fine. Is there something about video configuration that's permanent (ie, if we wanted to add it later we wouldn't be able to)?

Reply:
I don't know about video configuration that's permanent, but videos in their own levels sounds good!